File: src/tool/spring-boot-generator/output/contract/db/contractNotaryRecord.js
let contractNotaryRecord = {
	name: "contractNotaryRecord",
	description: "",
	columns: {
		contractId: {
			name: "contractId",
			description: "合同编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		notaryChannel: {
			name: "notaryChannel",
			description: "公证渠道",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		notaryEndTime: {
			name: "notaryEndTime",
			description: "公证完成时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		notaryStartTime: {
			name: "notaryStartTime",
			description: "公证发起时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		notaryStatus: {
			name: "notaryStatus",
			description: "公证状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		proposer: {
			name: "proposer",
			description: "公证申请人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		}
	}
};

let columnsArray = [
	"contractId",
	"createTime",
	"id",
	"notaryChannel",
	"notaryEndTime",
	"notaryStartTime",
	"notaryStatus",
	"proposer"
];

function create() {
  return {};
}

contractNotaryRecord.columnsArray=columnsArray;
contractNotaryRecord.create=create;
module.exports=contractNotaryRecord;

File: src/tool/spring-boot-generator/output/contract/contract/src/main/java/com/contract/pojo/entity/ContractNotaryRecord.java
package com.contract.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("")
@Getter
@Setter
public class ContractNotaryRecord  {

    @ApiModelProperty("合同编号")
    private Integer contractId;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("公证渠道")
    private Integer notaryChannel;

    @ApiModelProperty("公证完成时间")
    private Date notaryEndTime;

    @ApiModelProperty("公证发起时间")
    private Date notaryStartTime;

    @ApiModelProperty("公证状态")
    private Integer notaryStatus;

    @ApiModelProperty("公证申请人")
    private Integer proposer;

}
